VISION.md

```diff
@@ -103,6 +105,102 @@ The OpenEBS project and CNCF recognize the importance of supporting this and mai
+- **Dynamic Storage Provisioning**: Users can dynamically provision Persistent Volumes (PVs) with minimal configuration.
+- **Multiple Storage Engines**: OpenEBS offers various storage engines such as Local PV options (Hostpath, ZFS, LVM and Rawfile) and Replicated PV Mayastor, catering to different performance and resiliency requirements.
+- **Resiliency and Replication**: Provides synchronous and asynchronous replication for high availability.
+- **Cloud-Native Architecture**: Fully integrates with Kubernetes, leveraging its declarative nature and APIs for seamless storage operations.
+- **Observability and Monitoring**: Includes built-in monitoring with Prometheus and Grafana integrations.
+
+OpenEBS can be used in several cloud-native scenarios:
+
+### 1. Kubernetes Stateful Applications
+   - Supports applications such as MySQL, PostgreSQL, MongoDB, and Cassandra that require persistent storage.
+   - Ensures high availability and resilience through replication mechanisms.
+
+### 2. Hybrid and Multi-Cloud Storage
+   - Enables organizations to run stateful workloads across different cloud providers.
+   - Provides seamless storage portability, making migrations easier.
+
+### 3. Edge Computing and IoT
+   - Can be deployed in edge environments where lightweight, distributed storage solutions are needed.
+   - Works well with Kubernetes distributions optimized for edge computing (e.g., K3s, MicroK8s).
+
+### 4. CI/CD Pipelines and Dev/Test Environments
+   - Provides fast and ephemeral storage for continuous integration and testing workflows.
+   - Supports quick provisioning and teardown of test environments without affecting production systems.
+
+### 5. Disaster Recovery and Backup Solutions
+   - OpenEBS replication capabilities help create disaster recovery strategies for Kubernetes applications.
+   - Works with backup tools like Velero to create snapshots and restore data when needed.
+
+OpenEBS is a versatile cloud-native storage solution designed for Kubernetes workloads. By offering dynamic provisioning, multi-tenancy, and scalability, it bridges the gap between traditional storage and the growing needs of continuously evolving Kubernetes-based applications. OpenEBS is a compelling choice for enterprises looking to build resilient, portable, and high-performance storage solutions in diverse cloud-native environments.
+
+## Goals and Objectives
+
+OpenEBS is an open-source Cloud Native storage solution that provides container-attached storage for Kubernetes following vendor-neutral principles.  OpenEBS is a Kubernetes first approach to integrated granular storage management with data replication at infrastructure level, adhering to  Kubernetes-native principles, and delivering enhanced flexibility for stateful applications. OpenEBS empowers application owners to concentrate fully on their core application value. Storage resiliency and management becomes a worry free and reliable part of the underlying Kubernetes infrastructure.
```

Comment:
Seems a bit repeated to what's already said on Why OpenEBS

Reply:
Yes, I agree. Not sure how to fix :(

Reply:
Do we even need to have this, maybe just keep the goals as below?